### ISSUE TITLE
Something went wrong while saving the customer.

### ISSUE THREAD

**Issue by sivajik34** [magento/magento2#1437]
![customer_creation_from_admin](https://cloud.githubusercontent.com/assets/7636894/8493000/d23e65e8-2170-11e5-99f2-e82f7e2b7111.png)

**Comment by sivajik34** [comment]
![customer_creation_from_frontend](https://cloud.githubusercontent.com/assets/7636894/8493090/a1aec614-2172-11e5-9edc-0449c0c34b5b.png)

**Comment by vpelipenko** [comment]
@sivajik34, do you see any errors in log files?

**Comment by sivajik34** [comment]
its fine in merchant beta

**Comment by AzeemYusufali** [comment]
I am facing this issue with Magento v2.2.1. 

Since upgrading to this version, I cannot create customers in admin and am presented with "Something went wrong while saving the customer."

I do not know where to begin with resolving this issue. Any help would be greatly appreciated.

**Comment by roobware** [comment]
I have the same issue on Magento 2.2.1. My Version is upgraded from 2.1.6 to 2.2.1 via the composer. Magento is running with PHP 7.1.11 on Apache 2.4.25.

In my case will the validation break for 'DOB' input field. I have not filled out this value before.
The exception will come from the `'\Magento\Framework\Data\Form\Filter\Date'` class by the `'\Zend_Filter_NormalizedToLocalized->filter()'` function in function `inputFilter()`. The filter internal will be called with this parameter values: `'date_format' => 'yyyy-MM-dd'` and `'locale' => 'en_US'`.

In my case, i have added a simple `empty()` check in this core file to prevent this exception. But, this is a quick and dirty solution ;)

![image](https://user-images.githubusercontent.com/27640321/33203596-e0db313e-d100-11e7-9829-cb5a4c3bbd97.png)

**Comment by lano-vargas** [comment]
@sivajik34 https://github.com/magento/magento2/issues/13311

**Comment by Tr3Devil** [comment]
Hello, it all started when I decided to change the user from private to retailer. Then found the following problem after modifying the database "Customer_eav_attribute" changing in the column "input filter" the wording "datatime" in "data" as suggested by a friend.
Before the change I got an error in the string that I reported in the attachment
![prob utente ingrosso](https://user-images.githubusercontent.com/51789290/59436896-c3e8eb80-8df0-11e9-91f6-aa328baa645c.jpg)
![prob utente ingrosso2](https://user-images.githubusercontent.com/51789290/59436909-c9463600-8df0-11e9-9867-f84276972f07.jpg)
then I get the following error
![Modifica per problema ingrosso2](https://user-images.githubusercontent.com/51789290/59436925-d2370780-8df0-11e9-8ede-a2d32b51d97c.jpg)
![Modifica per problema ingrosso (2)](https://user-images.githubusercontent.com/51789290/59436985-f5fa4d80-8df0-11e9-8993-5dbcca489fc0.jpg)

How can I solve it? The change I made in the database must be made? There is someone who can suggest me a solution.
Thank you
